client/hooks/useCallbackRef.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * External dependencies
+ */
+import memize from 'memize';
+import { useCallback } from '@wordpress/element';
+
+function useCallbackRef( callback, deps ) {
```

Comment:
It looks like this was removed recently so we can remove this and copy the latest changes for `useFocusOnMount`

https://github.com/WordPress/gutenberg/blob/master/packages/compose/src/hooks/use-focus-on-mount/index.js